pkg/prometheus.go

```diff
@@ -41,6 +41,23 @@ var (
 		Name: "yace_cloudwatch_ec2api_requests_total",
 		Help: "Help is not implemented yet.",
 	})
+
+	replacer = strings.NewReplacer(
```

Comment:
Nit - use var blocks to group related variables. Replaces isn't related to the prometheus metrics, so put into it's own var block (or inline) outside of this one